mobile/src/main/res/layout/voice_widget.xml

```diff
@@ -10,12 +10,12 @@
         android:id="@+id/btn_mic"
 
         android:layout_centerHorizontal="true"
-        android:src="@drawable/menu_mic_dark"
+        android:src="@drawable/ic_mic_black_24dp"
```

Comment:
Please insert `android:tint="@color/fab_material_white"` here or any other white/grey color if you think thats suits better. But black is a bad color here.

Reply:
By adding the tint property, all goes crazy. The vector graphic get pixelatet. I'd rather keep the black icon.